maxim/logger/agno/client.py

```
@@ -0,0 +1,188 @@
+        id=str(uuid4()),
+        model=getattr(agent, "model", None),
+        provider="agno",
+        name=generation_name,
+        messages=[],
+        model_parameters={},
+    )
+    generation = trace.generation(gen_config)
+    return trace, generation, final_trace_id, is_local_trace
+
+
+def _log_event(trace: Any, generation: Any, event: Any, content: Any) -> None:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `trace`

(ANN401)